src/sentry/relay/config/__init__.py

```diff
@@ -589,8 +589,8 @@ def _filter_option_to_config_setting(flt: _FilterSpec, setting: str) -> Mapping[
 #: Version of the transaction metrics extraction.
 #: When you increment this version, outdated Relays will stop extracting
 #: transaction metrics.
-#: See https://github.com/getsentry/relay/blob/4f3e224d5eeea8922fe42163552e8f20db674e86/relay-server/src/metrics_extraction/transactions.rs#L71
-TRANSACTION_METRICS_EXTRACTION_VERSION = 2
+#: See https://github.com/getsentry/relay/blob/6181c6e80b9485ed394c40bc860586ae934704e2/relay-dynamic-config/src/metrics.rs#L85
+TRANSACTION_METRICS_EXTRACTION_VERSION = 3
```

Comment:
It will take a while for all project configs to update to version 3, so we will be in a state where Relay still emits outcomes (for rate limits) based on the old metric, but sentry already emits accepted outcomes based on the new metric. Can't think of an argument right now why this should matter, but it feels inconsistent.

Reply:
That's acceptable behavior, since both numbers are equal in practice.

Reply:
Should we continue accepting transaction duration outcomes for some time after upgrading the version to 3?